controllers/backstage_deployment.go

```diff
@@ -236,3 +231,11 @@ func getDefaultObjName(backstage bs.Backstage) string {
 func getDefaultDbObjName(backstage bs.Backstage) string {
 	return fmt.Sprintf("backstage-psql-%s", backstage.Name)
 }
+
+func setBackstageImage(deployment *appsv1.Deployment, imageName string) {
+	visitContainers(&deployment.Spec.Template, func(container *v1.Container) {
+		if container.Image == _defaultBackstageMainContainerName || container.Image == _defaultBackstageInitContainerName {
```

Comment:
container name, you mean, right?

```suggestion
		if container.Name == _defaultBackstageMainContainerName || container.Name == _defaultBackstageInitContainerName {
```